src/chess_teacher_stockfish/main.py

```diff
@@ -68,11 +81,12 @@ class AnalyzeMoveRequest(EvaluatePositionRequest):
 
 
 class EvaluatePositionResponse(CamelCaseAliasBaseModel):
-    """Response object containing the Stockfish evaluation of the current position and
-    the win/draw/loss statistics"""
+    """Response object containing the Stockfish evaluation of the current position,
+    the win/draw/loss statistics and the top three moves"""
 
     evaluation: dict[str, str | int]
     wdl_stats: list[int] | None
+    top_three_moves: list[TopMove] | None
```

Comment:
Line 111 states that `get_top_three_moves()` returns `List[TopMove]`, so in what case can it be `None`?